common/tasks.py

```diff
@@ -107,3 +116,24 @@ def pybabel_update(c):
 @task
 def pybabel_compile(c):
     c.run("pybabel compile -d translations")
+
+
+@task
+def full_bootstrap(c):
+    @contextmanager
+    def _env_var(key, value):
+        old_val = os.environ.get(key, "")
+        os.environ[key] = value
+        yield
+        os.environ[key] = old_val
+
+    with _env_var("FLASK_ENV", "development"):
+        with create_app().app_context():
+            alembic_cfg = Config("db/migrations/alembic.ini")
+            alembic_cfg.set_main_option("script_location", "db/migrations")
+            command.upgrade(alembic_cfg, "head")
+
+            load_all_fund_rounds()
+            load_fund_from_fab_impl()
```

Comment:
Should this pass `seed_all_funds=True` to the fab function?

Reply:
yes dang, good spot - thanks 🙏